lib/v2/screens/settings/edit_name/components/flat_button_long.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:meta/meta.dart';
```

Comment:
this button will be used in multiple places. 
Make generic and move to shared components.